packages/ember-routing/tests/system/router_test.js

```diff
@@ -25,42 +26,38 @@ QUnit.module("Ember Router", {
 
     // ensure rootURL is injected into any locations
     registry.injection('location', 'rootURL', '-location-setting:root-url');
-
-    Router = EmberRouter.extend();
   },
   teardown: function() {
     runDestroy(container);
-    registry = container = Router = router = null;
+    registry = container = null;
   }
 });
 
 test("can create a router without a container", function() {
-  Router.create();
+  createRouterWithContainer({ container: null });
 
   ok(true, 'no errors were thrown when creating without a container');
 });
 
 test("should not create a router.js instance upon init", function() {
-  createRouter();
+  var router = createRouterWithContainer();
 
   ok(!router.router);
 });
 
 test("should destroy its location upon destroying the routers container.", function() {
-  createRouter();
-
+  var router = createRouterWithContainer();
   var location = router.get('location');
 
-  run(container, 'destroy');
+  runDestroy(container);
```

Comment:
How about `seekAndDestroy`?

I don't think the `And` is necessary. The [helper](https://github.com/emberjs/ember.js/blob/e53f5709d05a872447c039048acaef03c0a3bfc6/packages/ember-runtime/tests/utils.js#L7-L11) was extracted by a popular pattern, and merely puts a name to

```js
run(function(foo) { foo.destroy(); });
```